site/docs/test/docker-from-release.yml

```
@@ -0,0 +1,12 @@
+- path: installation/docker.md
+  matches:
+    - get latest tag
+    - pull docker image
+    - download docker-compose
+    - setup environment
+    - create directories
+    - auto cleanup
+    - start service
+    - check wallet
+    - stop service
+    - cleanup
```

```suggestion
    - cleanup

```

(newline at end of file)